copylot/gui/defaults.txt

```diff
@@ -0,0 +1 @@
+{"parameters": {"exposure": [0.02, 0.001, 1.0], "nb_timepoints": [600, 1.0, 10000.0], "scan_step": [0.1, 0.01, 1.0], "stage_scan_range": [1000.0, 0.0, 10000.0], "vertical_pixels": [2048, 0.0, 4000.0], "num_samples": [20, 0.0, 100.0], "offset_view1": [1550.0, 0.0, 3180.0], "offset_view2": [1650.0, 0.0, 3180.0], "view1_galvo1": [4.2, -10.0, 10.0], "view1_galvo2": [-4.08, -10.0, 10.0], "view2_galvo1": [-4.37, -10.0, 10.0], "view2_galvo2": [3.66, -10.0, 10.0], "stripe_reduction_range": [0.1, 0.0, 10.0], "stripe_reduction_offset": [0.56, -10.0, 10.0]}, "live": [1, 0], "timelapse": [0, 2]}
```

Comment:
I thought we want to merge water dispenser PR first then JSON stuff, do we want to merge this txt with this PR?

Reply:
hmm, defaults.txt should be in gitignore... must have accidentally not deleted it

Reply:
alright, easy to address